magento-research/pwa-studio
# [docs]: Add federated search to docs

## Description
This PR adds the federated search feature to the docs site. You now can search for Devdocs, User Guide and OMS docs from PWA docs.

I would also like to commit Gemfile.lock file as we do on devdocs, since it's better way to manage the updates to theme and other gems this way.

## Verification Steps

1. Go to the docs site
2. Search for something
3. Or go to search results page.
4. See the search results from different sites. 

## How Have YOU Tested this?




## Screenshots / Screen Captures (if appropriate):

## Proposed Labels for Change Type/Package




## Checklist:
